src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/model/dao/LimitedPartnershipSubmissionDao.java

```diff
@@ -0,0 +1,44 @@
+package uk.gov.companieshouse.limitedpartnershipsapi.model.dao;
+
+import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.data.mongodb.core.mapping.Field;
+
+import java.time.LocalDateTime;
+
+@Document(collection = "limited_partnership_submissions")
+public class LimitedPartnershipSubmissionDao {
+
+    @Id
+    private String id;
+
+    @Field("created_at")
+    private LocalDateTime createdAt;
+
```

Comment:
should we have a created by user id here as well

Reply:
Possibly. I left it out as it's not on the data model. Is it a requirement?

Reply:
I would say it is, we've added it on previous projects, and can be used by support to identify who created the submission.

Reply:
Added